Gcpe.Hub.API/Controllers/MessagesController.cs

```diff
@@ -25,6 +25,24 @@ public MessagesController(HubDbContext dbContext, ILogger<MessagesController> lo
             this.mapper = mapper;
         }
 
+        // Bumps the sort order of all published messages from firstSortOrder to lastSortOrder
+        // A negative lastSortOrder omits it from the query
+        public static int BumpSortOrders(HubDbContext dbContext, int direction, int firstSortOrder, int lastSortOrder)

```

Comment:
If it's not static you can use the dbContext

Reply:
ah yes i originally had it somewhere else and static - will fix up